### ISSUE TITLE
zip File Support for GPX Tracks

### ISSUE THREAD

**Issue by sonora** [osmandapp/OsmAnd#19100]
Users using OsmAnd to organize and view a larger collection of GPX tracks face the situation that these are often hundreds, sometimes thousands of highly compressible text files,  which if handled as separate files
- in the file system eat up lots of space,
- and need lots of copying time when transferring to or from the device.

Would it perhaps be feasible to support GPX also when packaged in one or several zip files in OsmAnd's 'tracks' data folder?

**Comment by vshcherb** [comment]
Nope we plan to support OBF as collection which already possible to do and extend this support. We don't want to miss features or implement them twice

**Comment by sonora** [comment]
Understandable.

The drawback is that OBF is OsmAnd proprietary, while users using OsmAnd to handle their extensive GPX libraries would very likely welcome OsmAnd also handling some tool-independent efficient way of storing their file structure of GPX-standard files.

In particular since zipped GPX libraries are also the way e.g. commercial tour guides, hiking books or touring websites host and distribute these files.

**Comment by vshcherb** [comment]
Yes I totally understand, my vision would be to convert on the flight GPX.zip to OBF but as for now there is an issue that many functions won't work on Android of Map Creator.

Please not forget that mostly .zip is used for large collection of tracks and I don't want to implement features which only cause complaints that it doesn't smooth or correctly. Actually we need to introduce the limit only to be able display tracks < 1M points in total. That's still ongoing
